app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/activity/StripePaymentActivity.java

```diff
@@ -68,8 +73,13 @@ public void onCreate(Bundle savedInstanceState) {
 
         ticketPrice = getIntent().getStringExtra(Const.KEY_TICKET_PRICE);
         ticketHistory = getIntent().getIntExtra(Const.KEY_TICKET_HISTORY, -1);
+        termsOfServiceLink = getIntent().getStringExtra(Const.KEY_TERMS_LINK);
+        stripePublishableKey = getIntent().getStringExtra(Const.KEY_STRIPE_API_PUBLISHABLE_KEY);
```

Comment:
There needs to be some handling if the key is null. Either here or above. The payment activity should not proceed, if its empty.